server.js

```diff
@@ -9,6 +9,8 @@ var require = require(),
     io = require('socket.io');
 require('dotenv').config();
 
+require('dotenv').config();
```

Comment:
This was probably because I merged staging with my branch yesterday before pushing. I will do a rollback